### DISCUSSION TITLE
Language en_US missing on fresh install

### DISCUSSION THREAD

**Discussion by dononeil** [kimai/kimai#4602] [On-Premise: Installation & Updates]
I just did a fresh install today (with fresh GIT pull) of Kimai on my server to check it out, and after logging in for the first time it tries to go to URL: /en_US/done, if you go to a 'bad' page, it gives you the option to 'go home', which takes you to /en/dashboard/. When you logout, and log back in it tries to go /en_US/timesheet/. In the installation, there is no language file for en_US (there is en, en_CA, en_GB, etc...)

It looks like something is missing from the install... I tried copying en_GB to en_US but that didn't make a difference.

Fundamentally the install is 'broken'. How can I resolve this issue?

**Comment by kevinpapst** [comment]
What exactly is "broken"?

**Comment by dononeil** [comment]
When you login, and it tries to send you to /en_US/timesheet/ it causes a server error, and this is in the apache error log:

Wed Jan 31 09:31:28.343879 2024] [proxy_fcgi:error] [pid 8498:tid 140387901265664] (104)Connection reset by peer: [remote x.x.x.x:30760] AH01075: Error dispatching request to : , referer: https://time.XXXX.com/en_US/login

Similarly other references to en_US cause the same error.

**Reply by kevinpapst** [reply]
Can you share the error(s) from Kimai log (Doctor > Logs or var/log/prod.log or Docker logs).

It could be related to a recent change but I cannot reproduce that error.

**Comment by dononeil** [comment]
There isn't anything in the logfile... the error apache reports in chrome is error 503 Service Unavailable. Running PHP 8.1.24.  This only happens on URL's with the en_US in them like: en_US/timesheet/ 
and not: /en/doctor or en/dashboard/

Actually, I've been clicking on various links, and en/timesheet/, en/export/, en/team/timesheet/, en/admin/activity/, en/profile/Administrator, reports the same server error too. So it's like the install is damaged, or maybe the git repository is messed up??

Can you validate the repository is clean, and I can try a fresh install.

**Comment by dononeil** [comment]
I ran an update against the repository, and it said there were no changes.... so either something isn't compatible with my LAMP stack, or there is something wrong with the repository.

**Reply by kevinpapst** [reply]
Test the demo. If it works there, problem is in your environment
